smdebug/_version.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.19"
```

Comment:
will this change trigger a release?

Reply:
Yes @yl-to, once this change is merged into `master` it will trigger a pipeline to create a new pre-release pypi package for smdebug.  You can see [past pre-release examples here](https://pypi.org/project/smdebug/1.0.17b20220513/#history).  We do not plan to do a full sagemaker-debugger release here, but as you can see on that pypi page, a full release hasn't happened since version `1.0.12` anyway.

We will also need to add a new `1.0.20` tag to this repo for deep-learning-container builds.  The goal is for this updated pre-release sagemaker-debugger version to be used for creating new Sagemaker DLC images for the upcoming Tensorflow 2.10 release.  You can find an older example of how smdebug version `1.0.17` was used for TF 2.9 [here](https://github.com/aws/deep-learning-containers/blob/master/tensorflow/training/docker/2.9/py3/Dockerfile.cpu#L272).

Reply:
sounds okay, approving.